Firestore/Source/Core/FSTSyncEngine.h

```diff
@@ -40,17 +40,18 @@ namespace util = firebase::firestore::util;
 
 NS_ASSUME_NONNULL_BEGIN
 
-#pragma mark - FSTSyncEngineDelegate
-
-/**
- * A delegate to be notified when the client's online state changes or when the sync engine produces
- * new view snapshots or errors.
- */
-@protocol FSTSyncEngineDelegate
-- (void)handleViewSnapshots:(std::vector<core::ViewSnapshot> &&)viewSnapshots;
-- (void)handleError:(NSError *)error forQuery:(const core::Query &)query;
-- (void)applyChangedOnlineState:(model::OnlineState)onlineState;
-@end
+#pragma mark - SyncEngineCallback
+
+/** Defines an interface whose implementations can handle notifications from FSTSyncEngine. */
```

Comment:
How about taking the wording [from Android](https://github.com/firebase/firebase-android-sdk/blob/master/firebase-firestore/src/main/java/com/google/firebase/firestore/core/SyncEngine.java#L96):
```
Interface implemented by `EventManager` to handle notifications from `FSTSyncEngine`.
```
?